gallery/src/components/NxDropdown/NxDropdownLinksExample.tsx

```diff
@@ -17,17 +17,23 @@ function NxDropdownActionsExample() {
                 className="extra-class"
                 isOpen={isOpen}
                 onToggleCollapse={onToggleCollapse}>
-      <a href="https://www.google.com/" className="nx-dropdown-link disabled">
-        <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
+      <a href="https://www.google.com/" className="nx-dropdown-link">
         <span>Go somewhere</span>
+        <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
       </a>
       <a href="https://www.sonatype.com/" className="nx-dropdown-link">
-        <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
         <span>Go somewhere else</span>
+        <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
       </a>
-      <a href="https://developer.mozilla.org/en-US/docs/Web" className="nx-dropdown-link">
+      <a href="https://www.google.com/" className="nx-dropdown-link disabled">
```

Comment:
The disabled links should actually be functionally disabled.  The best way to do that is with an `onClick` handler that calls `preventDefault()` on the event object.

Reply:
>The disabled links should actually be functionally disabled. The best way to do that is with an onClick handler that calls preventDefault() on the event object.

updated here: [2942937](https://github.com/sonatype/sonatype-react-shared-components/pull/770/commits/2942937bd4601eb09dee4e6ddae38a67fcef7901)